package/CHANGELOG

```diff
@@ -27,6 +27,8 @@ Enhancements
   * Added a `exclude_same` kwarg to InterRDF (PR #4161)
   * LAMMPSDump Reader use of `continuous` option to stitch trajectories
     (Issue #3546)
+  * Add support for reading chainID info from prmtop amber topologies
+	(PR #4007)
```

Comment:
Still not resolved because your editor inserted a `<TAB>` instead of spaces, try
```suggestion
  * Add support for reading chainID info from prmtop amber topologies
    (PR #4007)
```